libs/langgraph/langgraph/_internal/_constants.py

```diff
@@ -77,6 +78,8 @@
 # key for the configurable dict in RunnableConfig
 NULL_TASK_ID = sys.intern("00000000-0000-0000-0000-000000000000")
 # the task_id to use for writes that are not associated with a task
+RUNTIME_PLACEHOLDER = "__pregel_runtime_placeholder__"
```

Comment:
Could we rename this to be specific to untracked values? ("runtime" is overloaded)